jujugui/static/gui/src/test/test_app.js

```diff
@@ -1488,6 +1571,7 @@ describe('App', function() {
         jujuCoreVersion: '2.0.0',
         user: userClass
       });
+      app.controllerAPI.connect();
```

Comment:
The connect method sets up the websocket object, which is expected to exist in a few of the tests--specifically those determining if the login form should display based on gisf status and the status of the apis.